Source/Scene/MetadataClass.js

```diff
@@ -72,7 +69,6 @@ Object.defineProperties(MetadataClass.prototype, {
    * @memberof MetadataClass.prototype
    * @type {Object.<String, MetadataClassProperty>}
    * @readonly
-   * @private
    */
   propertiesBySemantic: {
```

Comment:
`propertiesBySemantic` can stay `@private` since it's just used internally as an optimization